### PR TITLE
Added release notes for 9/28

### PR DESCRIPTION
**Pull request by jwitz**
Release notes and changes to docs based on https://github.com/astronomer/astrohub-hq-manifests/pull/734\

CLI-based changes are available to review here: https://github.com/astronomer/cloud-docs/pull/108

Should I include reference to the end of private beta? Only allusion right now is in the docs password change. Not sure how we want to promote that in general

### CODE REVIEW

File: docs/configure-deployment.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To create an Airflow Deployment on Astronomer Cloud:
 2. On the top right-hand side of the Deployments page, click **New Deployment**.
 3. Set the following:
     - **Name**
-    - **Astronomer Runtime**: For Private Beta, Astronomer Runtime 2.1.1 (based on Airflow 2.1.1) is hard-coded in this view and does not reflect your true running version.
+    - **Astronomer Runtime**
```
[the remark this answers is not in the document]
Comment (paolaperaza) [reply docs/configure-deployment.md:25]: Hm, not sure what you mean. Latest Runtime version (patch specified) is selected by default when you create a new Deployment. In this doc I'd just say, "By default, latest version of Astronomer Runtime is selected."

image

File: docs/overview.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Astronomer Cloud's architecture enables a few key features, available in Private
 - A secure control plane managed by Astronomer
 - Support for a multi-tenant data plane hosted in your organization's network on AWS (`us-east-1` and `us-west-2`)
 - Worker auto-scaling, powered by Airflow's Celery Executor + KEDA
-- Astronomer Runtime, a new collection of Docker images set to provide a differentiated Airflow experience. Astronomer Runtime initially supports Airflow 2.1.1
+- Astronomer Runtime, a new collection of Docker images set to provide a differentiated Airflow experience
```
Comment (paolaperaza) [inline docs/overview.md:23]:
```suggestion
- Astronomer Runtime, a new collection of Docker images set to provide a differentiated Airflow experience. Astronomer Runtime currently supports Airflow 2.1.1. Support for Airflow 2.2 coming soon.
```

File: docs/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,17 @@ If you have any questions or a bug to report, don't hesitate to reach out to us
...
+## September 28, 2021
+
+### Minor Improvements
+
+- In the UI, a new element on the Deployment information screen shows the health status of a Deployment. Currently, a Deployment is considered unhealthy if its Webserver is down.
+- You no longer need a password to access Astronomer Cloud documentation.
+
+### Bug Fixes
+
+- Fixed an issue where the UI Deployment information screen showed an incorrect Astronomer Runtime version number
```
Comment (paolaperaza) [inline docs/release-notes.md:25]: This was a whole feature! Let's throw this in improvements.

File: docs/release-notes.md [hunk 4]
```diff
@@ -13,6 +13,17 @@ If you have any questions or a bug to report, don't hesitate to reach out to us
 **Latest Runtime Version**: 3.0.3 ([Release notes](runtime-release-notes))
 **Latest CLI Version**: 0.2.9-beta ([Release notes](cli-release-notes))
 
+## September 28, 2021
+
+### Minor Improvements
+
+- In the UI, a new element on the Deployment information screen shows the health status of a Deployment. Currently, a Deployment is considered unhealthy if its Webserver is down.
+- You no longer need a password to access Astronomer Cloud documentation.
```
Comment (paolaperaza) [inline docs/release-notes.md:21]:
```suggestion
- In the Astronomer UI, a new element on the Deployment information screen shows the health status of a Deployment. Currently, a Deployment is considered unhealthy if the Airflow Webserver is not running and the Airflow UI is not available.
- We've moved our documentation home for Astronomer Cloud to `docs.astronomer.io`, and you no longer need a password to access the page.
```

File: docs/release-notes.md [hunk 5]
```diff
@@ -13,6 +13,17 @@ If you have any questions or a bug to report, don't hesitate to reach out to us
 **Latest Runtime Version**: 3.0.3 ([Release notes](runtime-release-notes))
 **Latest CLI Version**: 0.2.9-beta ([Release notes](cli-release-notes))
```
Comment (paolaperaza) [inline docs/release-notes.md:14]:
```suggestion
**Latest CLI Version**: 1.0.0 ([Release notes](cli-release-notes))
```

File: docs/release-notes.md (hunk 3 above)
Comment (paolaperaza) [inline docs/release-notes.md:25]:
```suggestion
- The Astronomer UI now correctly renders a Deployment's running version of Astronomer Runtime. This version is verified every time you deploy code and is updated in real-time.
```

File: docs/known-limitations.md [hunk 6]
```diff
@@ -15,7 +15,6 @@ The list below reflects known issues at the time of writing. It will be updated
 - Assistance from our team is required to give the first user in your Organization access to Astronomer Cloud.
 - When a user first creates an account, they will be asked to validate their email address. Email validation is not currently required to access Astronomer Cloud, but we encourage users to follow the process anyway as we will enforce it in the future.
 - On Private Beta, your team will be limited to a single Astronomer Workspace that supports multiple users and Deployments. Support for an Astronomer "Organization" with multiple Workspaces coming soon.
-- Multiple versions of Astronomer Runtime are supported, but the Astronomer UI is hard-coded to show version 2.1.1 and will not reflect your actual running version. Version awareness in the Astronomer UI, API, and CLI is coming soon.
 - We do not currently support PgBouncer, but the RDS instance provisioned in your Cluster will support around 1000 connections to your database, enough to support 10-12 Deployments.
```
Comment (paolaperaza) [inline docs/known-limitations.md:18]:
```suggestion
- If you're running Astronomer Runtime `2.1.1`, `3.0.0`, or `3.0.1`, the Astronomer Runtime field in the Astronomer UI will show `Unknown`. Once you upgrade to Runtime 3.0.2+, your Deployment's version of Runtime will be correctly listed.
- We do not currently support PgBouncer, but the RDS instance provisioned in your Cluster will support around 1000 connections to your database, enough to support 10-12 Deployments.
```

**Review by paolaperaza — COMMENTED** [review]
Some comments here based on my understanding of our changes, @jwitz! 

cc: @adam2k Feel free to take a look here real quick to make sure we're not missing anything.

File: docs/release-notes.md (hunk 4 above)
Comment (paolaperaza) [reply docs/release-notes.md:21]: Is this worth a screenshot maybe? We don't typically do that in release notes, but I don't have a strong reason for why we wouldn't? 🤷‍♀️ 

(I suppose if we're linking out to another topic-specific doc that has a screenshot)

File: docs/release-notes.md (hunk 3 above)
Comment (jwitz) [reply docs/release-notes.md:31]: Removing the second sentence, as that conclusion follows clearly from the first sentence alone. Also sounds a bit enhancey when this was very much a bug from the start.

**Comment by jwitz** [comment]
@paolaperaza Updated! Any comments about CLI-related content can now go here: https://github.com/astronomer/cloud-docs/pull/108

File: docs/configure-deployment.md [hunk 7]
```diff
@@ -22,8 +22,8 @@ To create an Airflow Deployment on Astronomer Cloud:
 2. On the top right-hand side of the Deployments page, click **New Deployment**.
 3. Set the following:
     - **Name**
-    - **Astronomer Runtime**: For Private Beta, Astronomer Runtime 2.1.1 (based on Airflow 2.1.1) is hard-coded in this view and does not reflect your true running version.
     - **Description**
+    - **Astronomer Runtime**: By default, the latest version of Astronomer Runtime is selected
```
Comment (paolaperaza) [inline docs/configure-deployment.md:26]:
```suggestion
    - **Astronomer Runtime**: By default, the latest version of Astronomer Runtime is selected.
```

There is a period on other sentences?

File: docs/release-notes.md [hunk 8]
```diff
@@ -11,7 +11,29 @@ Astronomer is committed to continuous development during the Private Beta Progra
...
 
 **Latest Runtime Version**: 3.0.3 ([Release notes](runtime-release-notes))
-**Latest CLI Version**: 0.2.9-beta ([Release notes](cli-release-notes))
+**Latest CLI Version**: 1.0.0 ([Release notes](cli-release-notes))
+
+## September 28, 2021
+
+:::danger
+
+This feature introduces a breaking change to code deploys. Starting on September 28, you must be using version 1.0.0 of the Astronomer CLI to successfully deploy code onto any Deployment. For more information, read the [CLI release notes](cli-release-notes).
```
Comment (paolaperaza) [inline docs/release-notes.md:20]:
```suggestion
This release introduces a breaking change to code deploys via the Astronomer CLI. Starting on September 28, you must upgrade to v1.0.0 of the CLI to deploy code to Astronomer. [CI/CD processes](ci-cd) enabled by Deployment API Keys will continue to work and will not be affected. For more information, read the [CLI release notes](cli-release-notes).
```

**Review by paolaperaza — APPROVED** [review]
Minor comments here but LGTM. Are we sure we want to keep the Runtime version under "bug fix"?

File: docs/release-notes.md (hunk 3 above)
Comment (paolaperaza) [reply docs/release-notes.md]: We certainly weren't talking about it as a bug internally! Took a lot of work to address and version awareness is seen as a value-add that will unlock a lot of other cool things. Yes for a managed Airflow service this is an expected "given", but I feel like it's not a big deal to have this under `Minor Improvements`?

**Comment by jwitz** [comment]
@paolaperaza The Runtime version change is for sure a bug fix. If we didn't previously include a UI element showing Runtime version, it might have been an improvement. But we were showing 2.1.1 for a long time, which is not expected behavior beyond the fact that we put it in Known Limitations.

I'll wait to merge this until somebody's reviewed https://github.com/astronomer/cloud-docs/pull/108

**Pull request merged**